### ISSUE TITLE
load color schemes from external image files

### ISSUE THREAD

**Issue by HeladoDeBrownie** [HeladoDeBrownie/Nexus#51]
should be able to have files with seven pixels in a row that correspond to the seven colors of a color scheme and be able to load those instead of textually specifying color schemes

**Comment by HeladoDeBrownie** [comment]
wait, no, an image file wouldn't work because i use a different color format. disregard lmao
